### PR TITLE
History

### PR DESCRIPTION
**Pull request by malsbat**
Outgoing publication history

### CODE REVIEW

File: inc/dps/dps.h [hunk 1]
```diff
@@ -776,6 +776,17 @@ DPS_Status DPS_InitPublication(DPS_Publication* pub,
                                const DPS_KeyId* keyId,
                                DPS_AcknowledgementHandler handler);
 
+/**
+ * Configure the Quality-of-Service (QoS) parameters of a publication.
```
Comment (GregBurns) [inline inc/dps/dps.h]: Maybe define a QoS struct that can be extended if new QoS capabilities are added in the future?

File: test/publish.c [hunk 2]
```diff
@@ -161,14 +162,256 @@ static void TestLoopbackAckLargeMessage(DPS_Node* node)
     DPS_DestroyPublication(pub);
 }
 
+#define HISTORY_CAP 10
```
Comment (GregBurns) [inline test/publish.c]: Have you tested out of order arrival of ACKs?

File: inc/dps/dps.h (hunk 1 above)
Comment (malsbat) [reply inc/dps/dps.h]: Yes, I'll do this

File: test/publish.c (hunk 2 above)
Comment (malsbat) [reply test/publish.c]: No, but should be easy to add

**Pull request merged**